app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -20,13 +20,16 @@ def show
     end
 
     def update
-      form = InProgressForm.where(form_id: params[:id], user_uuid: @current_user.uuid).first_or_initialize
+      form = InProgressForm.where(form_id: params[:id], user_uuid: @current_user.uuid).first
+      alt_id = "#{InProgressForm::ACCT_ID_PREFIX}#{@current_user.account_id}"
```

Comment:
How about making a method for `alt_id` in `User`?  Looks like there are quite a few places where these two strings are cobined.